lib/workers/repository/config-migration/pr/index.ts

```diff
@@ -0,0 +1,108 @@
+import is from '@sindresorhus/is';
+import { GlobalConfig } from '../../../../config/global';
+import type { RenovateConfig } from '../../../../config/types';
+import { logger } from '../../../../logger';
+import { platform } from '../../../../modules/platform';
+import { emojify } from '../../../../util/emoji';
+import { deleteBranch } from '../../../../util/git';
+import * as template from '../../../../util/template';
+import {
+  addAssigneesReviewers,
+  getPlatformPrOptions,
+  prepareLabels,
+} from '../../update/pr';
+import { getMigrationBranchName } from '../common';
+import { getErrors, getWarnings } from './errors-warnings';
+
+export async function ensureConfigMigrationPr(
+  config: RenovateConfig
+): Promise<void> {
+  if (!config.configMigration) {
+    return;
+  }
+  logger.debug('ensureConfigMigrationPr()');
+  logger.trace({ config });
+  const branchName = getMigrationBranchName(config);
+  const existingPr = await platform.getBranchPr(branchName);
+  logger.debug('Filling in config migration PR template');
+  let prTemplate = `Config migration needed, merge this PR to update your Renovate configuration file.\n\n`;
+  prTemplate += emojify(
+    `
+
+
+---
+{{{warnings}}}
+{{{errors}}}
```

Comment:
add an alternative message when no errors or warning?can that happen?

Reply:
Added

https://github.com/ladzaretti/feat-config-migration/pull/1